labgrid/driver/sshdriver.py

```diff
@@ -101,9 +101,9 @@ def _check_master(self):
     @Driver.check_active
     @step(args=['cmd'], result=True)
     def run(self, cmd, codec="utf-8", decodeerrors="strict", timeout=None): # pylint: disable=unused-argument
-        return self._run(cmd, codec=codec, decodererrors=decodeerrors)
+        return self._run(cmd, codec=codec, decodeerrors=decodeerrors)
```

Comment:
Shouldn't we pass on the timeout value to `_run()` here?

Reply:
Timeout is actually ignored. commandmixin just happen to pass it to sshdriver.
The new is that commandmixin now calls _run instead of run, so actually it could be removed here, but that would not be backwards compatible.